lib/string_foundation/symbol.rb

```diff
@@ -0,0 +1,13 @@
+# ==============================================================================
+# LIB - STRING FOUNDATION - SYMBOL
```

Comment:
Change the name from The Symbol Methods to The Is Methods, so rename this to `is.rb` .